src/colorprint.cc

```diff
@@ -136,13 +136,16 @@ void ColorPrintf(std::ostream& out, LogColor color, const char* fmt,
   CONSOLE_SCREEN_BUFFER_INFO buffer_info;
   GetConsoleScreenBufferInfo(stdout_handle, &buffer_info);
   const WORD old_color_attrs = buffer_info.wAttributes;
-
+  WORD  new_attr;
+  
   // We need to flush the stream buffers into the console before each
   // SetConsoleTextAttribute call lest it affect the text that is already
   // printed but has not yet reached the console.
   fflush(stdout);
+  new_attr = (buffer_info.wAttributes & ~7);
+  new_attr &= ~8;    // Since 'wAttributes' could have been hi-intensity at startup.
```

Comment:
What are 7 and 8 here? Can we avoid magic numbers?

Reply:
Is https://docs.microsoft.com/en-us/windows/console/console-screen-buffers#character-attributes lying and there are no macros for these magic numbers?

Reply:
I give up.

Reply:
the comment suggests there's a bug here. `BACKGROUND_INTENSITY` is 0x0080 i think. `FOREGROUND_INTENSITY` is 0x0008.

same with the masking of ~7 if you're trying to mask off the backgrounds.. not sure how this works (i am not a windows dev though).

Reply:
Let me rephraze: can you not just replace `8` with `BACKGROUND_INTENSITY` and `7` with `FOREGROUND_INTENSITY`?